doc/user_documentation.rst

```diff
@@ -144,6 +144,18 @@ It's even easier for a Fedora Rawhide package::
 because 'fedora' distgit is the default, and we automatically pick the default
 branch.
 
+.. note::
+    Please note that an SRPM is downloaded from the specified DistGit
+    only once per Copr build, regardless of the number of enabled
+    chroots. It **is not** the case that within one Copr build,
+    ``fedora-37-x86_64`` chroot would be built from the ``f37``
+    branch, ``fedora-38-x86_64`` from the ``f38`` branch, and
+    ``fedora-39-x86_64`` from the ``rawhide`` branch. You can,
```

Comment:
I know what you try to say by the `It is not ...` part, but it might be hard for those who don't know the behavior.
What about:
```
.. enabled chroots.  The effect is that the sources selected (e.g. Fedora `rawhide` DistGit branch) can easily be incompatible with the target chroot (e.g. `EPEL`, different dependencies, build tooling, etc.).
```

Reply:
I kinda think what you are saying and what I am saying, are different but both important things. I kept my explanation but added one more paragraph.

What do you think?

Reply:
If this isn't a duplicate to the part `The sources from your DistGit branch...` below, then I don't get what you are saying!  :-)  We can discuss this on monday..  might be an issue on my radio, I admit.